src/services/DataService.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  AirImpurityDataClass,
+  type SensorData,
+  SensorDataClass
+} from '@/types/models/SensorData';
+
+const locationsURI = 'http://localhost:8080/sampleData';
```

Comment:
This URL should be placed in the `.env` file. Maybe not the entire url, but only the part with localhost and port. You should create a `.env` file in the project root directory and create an environmental variable, e.g.:
```LOCAL_API=http://localhost:8080```

Then, you will need to install the additional package called `dotenv`:
```yarn add dotenv```

And finally add this import:
```ts
import dotenv from 'dotenv';
```
and this line of code:
```ts
dotenv.config();
```
to the `src/index.tsx` file.

The last step will be replacing the hardcoded part of the `locationsURI` with:
```ts
${process.env.LOCAL_API as string}
```